src/components/Dialog/ButtonClose/ButtonClose.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {block} from '../../utils/cn';
+import {Button} from '../../Button';
+import {Icon} from '../../Icon';
+import {CrossIcon} from '../../icons/CrossIcon';
+import './ButtonClose.scss';
+
+const b = block('dialog-btn-close');
+
+export interface ButtonCloseProps {
+    handleClose: (
```

Comment:
props should be named `on*`, inner handlers - `handle*`

Reply:
Fixed!